pages/peterportal/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+import { Steps } from "nextra/components";
+
+# Getting Started
+
+This document will guide you through setting up the project on your local machine, from cloning the repository to running the project locally. Whether you're a committee member or an open-source contributor, you'll find all the necessary steps to get started below.
+
+## Setup
+
+<Steps>
+
+### Cloning the Repository
+
+#### For Committee Members
+
+1. Clone the repository using the following command:
+   `git clone https://github.com/icssc/peterportal-client`
```

Comment:
I think some of these commands would look a better as multi-line comments. That way they're all on one line and easier to copy/paste. Right now, some of them wrap to the next line:
![image](https://github.com/icssc/docs/assets/8922227/6a701436-1522-4515-b434-a34d7ffd6df2)

Reply:
You can also get syntax highlighting if you use a code block, rather than inline:

```bash
git clone https://github.com/icssc/Zotistics.git
cd Zotistics
```